cli/tools/info.rs

```diff
@@ -165,8 +166,12 @@ fn print_cache_info(
 
 fn add_npm_packages_to_json(
   json: &mut serde_json::Value,
-  npm_resolver: &CliNpmResolver,
+  npm_resolver: &dyn CliNpmResolver,
 ) {
+  let Some(npm_resolver) = npm_resolver.as_managed() else {
+    return; // does not include byonm to deno info's output
```

Comment:
Could you explain why we don't want to include that in `deno info`?

Reply:
It would be complex. I'm not sure it's worth the effort until someone wants it. At the moment it just shows the entrypoints into the node_modules folder.